scripts/dayenu_delay_filter_run_baseline_parallelized.py

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# Copyright 2019 the HERA Project
+# Licensed under the MIT License
+
+"Command-line drive script for hera_cal.delay_filter with baseline parallelization. Only performs filtering for DAYENU"
+
+from hera_cal import delay_filter
+import sys
+
+parser = delay_filter.delay_filter_argparser(mode='dayenu', multifile=True)
+a = parser.parse_args()
+
+# set kwargs
+filter_kwargs = {'standoff': a.standoff, 'horizon': a.horizon, 'tol': a.tol,
+                 'skip_wgt': a.skip_wgt, 'min_dly': a.min_dly}
+baseline_list = io.baselines_from_filelist_position(filename=a.infilename, filelist=a.datafilelist)
+outfilename = a.res_outfilename
+spw_range = a.spw_range
+# allow none string to be passed through to a.calfile
+if a.calfile_list is not None:
```

Comment:
Probably better to check whether it's a string.

Reply:
i do this now.